### PR TITLE
Improve controller widget

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: packages/base/src/widget.ts [hunk 1]
```diff
@@ -745,6 +745,45 @@ class JupyterPhosphorWidget extends Widget {
...
+        }
+        super.dispose();
+        if (this._view) {
+            this._view.remove();
+        }
+        this._view = null;
+    }
+
+    private _view: DOMWidgetView;
+}
```
Comment (SylvainCorlay) [inline packages/base/src/widget.ts]: Moved `JupyterPhosphorPanelWidget` from controls to base next to `JupyterPhosphorWidget`.

**Comment by SylvainCorlay** [comment]
@jasongrout this is ready to go.

Note the move of `JupyterPhosphorPanelWidget` to base.

**Comment by SylvainCorlay** [comment]
Discussed the PR with @jasongrout. We need to include the move to JupyterPhosphorPanelWidget to base in the release notes.

**Comment by vidartf** [comment]
This seems ok, but could you add an description for why this was needed?

**Pull request merged**